app/src/main/java/com/example/habitsmasher/listeners/ClickListenerForFollowers.java

```diff
@@ -0,0 +1,43 @@
+package com.example.habitsmasher.listeners;
+
+import androidx.fragment.app.Fragment;
+import android.os.Bundle;
+import android.view.View;
+
+import androidx.navigation.NavController;
+import androidx.navigation.fragment.NavHostFragment;
+
+import com.example.habitsmasher.R;
+
+public class ClickListenerForFollowers implements View.OnClickListener{
+    /**
+     * This listener is used for the followers button, and leads to the list of people that
```

Comment:
Do we need to have two separate listeners for following and followers? Maybe pass in a variable that specifies which one it is.